listenbrainz/webserver/static/js/src/Bar.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react";
+import { ResponsiveBar, LabelFormatter } from "@nivo/bar";
+
+export type BarProps = {
+  data: Array<object>;
+  maxValue: number;
+};
+
+export type BarState = {
+  marginLeft: number;
+};
+
+export default class Bar extends React.Component<BarProps, BarState> {
+  constructor(props: BarProps) {
+    super(props);
+
+    this.state = {
+      marginLeft: window.innerWidth / 5,
+    };
+  }
+
+  componentDidMount() {
+    window.addEventListener("resize", this.handleResize);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener("resize", this.handleResize);
+  }
+
+  handleResize = () => {
+    this.setState({
+      marginLeft: window.innerWidth / 5,
+    });
+  };
+
+  render() {
+    const { data, maxValue } = this.props;
+    const { marginLeft } = this.state;
+
+    const leftAlignedTick = (tick: any) => {
```

Comment:
Can we fix this `any`?

Reply:
`nivo` itself doesn't provide type definition for the `tick` variable, I will add a definition for it using `console.log` statement although it might change in the future.